test/fw/ptl/utils/pbs_snaputils.py

```diff
@@ -1690,8 +1691,7 @@ def capture_system_info(self):
             win_platform = True
 
         # Capture information that's dependent on commands
-        for (key, values) in self.sys_info.iteritems():
-            sudo = False
+        for (key, values) in self.sys_info.items():
```

Comment:
why you removed `sudo = False` from next line?

Reply:
Added back.